src/unstable/components/Form/widgets/PasswordWidget.tsx

```diff
@@ -48,25 +54,24 @@ const PasswordStrength = ({ password }: PasswordStrengthProps) => {
 		if (zxcvbn && password) {
 			try {
 				const { score } = zxcvbn(password);
-				setStrengthScore(score);
+				setStrengthTitle(STRENGTH_TITLES[score]);
```

Comment:
Is this refactor required due to the Typescript bump?

Reply:
yes @LucianBuzzo, but @sradevski helped me find a much better solution